torch/_inductor/codecache.py

```diff
@@ -1593,9 +1599,10 @@ def split_aot_inductor_output_path(path: str) -> Tuple[str, str]:
         return path, ""
 
 
+@clear_on_fresh_inductor_cache
 class CudaKernelParamCache:
     cache: Dict[str, Dict[str, str]] = dict()
-    clear = staticmethod(cache.clear)
+    cache_clear = staticmethod(cache.clear)
```

Comment:
In order to use the new decorator, the clear method must be called `cache_clear`. I assume it's ok to change the names since I found no references in the existing code base; nothing here is a public API right?

Reply:
This is under _inductor, everything here is private api